lib/vsc/utils/fancylogger.py

```diff
@@ -443,15 +450,39 @@ def logToScreen(enable=True, handler=None, name=None, stdout=False):
 
     by default, logToScreen will log to stderr; logging to stdout instead can be done
     by setting the 'stdout' parameter to True
+
+    The `color` parameter enables or disables log colorization using
+    ANSI terminal escape sequences, according to the following values:
+
+    * when `color` is ``auto`` or ``None`` (default), then try to
+      auto-detect whether the output stream is connected to a terminal;
+    * when `color` is ``True`` or the string ``'yes'``, then turn on
```

Comment:
Well, since we do not have a natural way of doing a Yes/No/Maybe enum in Python, I thought that 'auto'/'yes'/'no' would do, but then True/False makes also a lot of sense from an API point of view.

I do not have a strong opinion here, so just suggest what triple of values you would like to use and I'll make the function accept that.